Forta-Bots-Challenge-3/README.md

```diff
@@ -0,0 +1,27 @@
+# MakerDAO Bridge Invariant Bot
+
+## Description
+
+This bot detects when the MakerDAO bridge invariant is violated on Arbitrum and Optimism
+
+## Supported Chains
+
+- Ethereum
+- Arbitrum
+- Optimism
+
+## Alerts
+
+- NETHERMIND-1
+  - Fired when the supply invariant of the MakerDAO Bridge is violated on Optimism or Arbitrum
+  - Severity is always set to "low"
+  - Type is always set to "info"
```

Comment:
I would say, given the nature of what this bot is monitoring, it would be of both higher severity and type.